Config/BuildSettings.swift

```diff
@@ -102,6 +102,9 @@ final class BuildSettings: NSObject {
     static let serverConfigDefaultHomeserverUrlString = "https://matrix.org"
     static let serverConfigDefaultIdentityServerUrlString = "https://vector.im"
     
+    // Force the user to set a homeserver instead of using the default one
+    static let forceHomeserverSelection = false
```

Comment:
you should move this flag before the definition of the default homeserver url (By this way the developer would know that the default homeserve url is ignored, so he should don't care of this url value)

What is the situation about the default Identity server? is it optional? This default url is not used anymore on authentication screen, we should update the comment here. If i'm correct it is used in Settings when the user decides to enable the discovery option

Reply:
@giomfo, ok I moved this new flag on the top of the `server configuration` section.
Concerning the default identity server, it's perfectly fine to set an empty string to `serverConfigDefaultIdentityServerUrlString`